### PR TITLE
docs(config) document devServer sockPort option

### PR DESCRIPTION
**Pull request by EugeneHlushko**
Takes one off #2974

Documents devServer `sockPort`

ON HOLD due to waiting dev server release.

### CODE REVIEW

File: src/content/configuration/dev-server.md [hunk 1]
```diff
@@ -1192,6 +1192,23 @@ module.exports = {
 };
 ```
 
+## `devServer.sockPort`
+
+`number` `string`
```
Comment (hiroppy) [inline src/content/configuration/dev-server.md]: Maybe we should add null?

https://github.com/webpack/webpack-dev-server/blob/master/lib/options.json#L239

Comment (EugeneHlushko) [reply src/content/configuration/dev-server.md]: `null` doesn't affect anything, it should be removed from the schema imho:
https://github.com/webpack/webpack-dev-server/blob/cf4d0d07b8ffba9d28cd627b48f74e5f4f307fc6/lib/utils/addEntries.js#L20

**Comment by hiroppy** [comment]
We have released webpack-dev-server@3.4.1, so I remove `on hold` label. /cc @EugeneHlushko

**Comment by EugeneHlushko** [comment]
Thanks everyone involved!

**Pull request merged**